src/commands/update.ts

```diff
@@ -62,32 +67,34 @@ export default class UpdateCommand extends Command {
     if (flags.available) {
       const {distTags, index, localVersions} = await lookupVersions(updater, this.config)
 
-      const headers = [
-        {align: 'left', value: 'Location'},
-        {align: 'left', value: 'Version'},
-      ]
-
-      if (distTags) {
-        headers.push({align: 'left', value: 'Channel'})
-      }
-
-      // eslint-disable-next-line new-cap
-      const t = TtyTable(
-        headers,
-        sort(Object.keys(index))
-          .reverse()
-          .map((version) => {
-            const location = localVersions.find((l) => basename(l).startsWith(version)) || index[version]
-            if (distTags) {
-              return [location, version, distTags[version] ?? '']
-            }
-
-            return [location, version]
-          }),
-        {compact: true},
-      )
-
-      ux.stdout(t.render())
+      const data = Object.keys(index).map((version) => {
+        const location = localVersions.find((l) => basename(l).startsWith(version)) || index[version]
+        const channel =
+          distTags[version] === 'latest'
+            ? 'stable'
+            : distTags[version] === 'latest-rc'
+              ? 'stable-rc'
+              : distTags[version]
+        return {
+          channel,
+          downloaded: location.includes('http') ? '' : 'true',
+          location,
+          version: this.config.version === version ? `${ux.colorize('yellowBright', version)} (current)` : version,
+        }
+      })
+
+      printTable({
+        borderStyle: 'headers-only-with-underline',
```

Comment:
I think the columns look sloppy since most of them are empty. I think `vertical-with-outline` looks a little cleaner